tests/meta/upload/conftest.py

```diff
@@ -96,29 +97,57 @@ def fixture_genotype_analysis_sex() -> dict:
     return {"ADM1": "male", "ADM2": "male", "ADM3": "female"}
 
 
-@pytest.fixture(name="mip_case")
-def fixture_mip_case(dna_mip_context, helpers) -> Family:
-    """Return a MIP case."""
+@pytest.fixture(name="mip_dna_case")
+def fixture_mip_dna_case(mip_dna_context, helpers) -> Family:
+    """Return a MIP DNA case."""
 
-    store = dna_mip_context.status_db
+    store = mip_dna_context.status_db
 
-    mip_case = helpers.add_case(
+    mip_dna_case = helpers.add_case(
         store=store,
-        internal_id="mip-case",
-        name="mip-case",
+        internal_id="mip-dna-case",
+        name="mip-dna-case",
         data_analysis=Pipeline.MIP_DNA,
     )
-    mip_sample = helpers.add_sample(
+    dna_mip_sample = helpers.add_sample(
         store,
-        internal_id="mip-case",
+        internal_id="mip-dna-case",
         application_type="wgs",
     )
-    helpers.add_relationship(store, case=mip_case, sample=mip_sample)
+    helpers.add_relationship(store, case=mip_dna_case, sample=dna_mip_sample)
 
     helpers.add_analysis(
         store,
-        case=mip_case,
+        case=mip_dna_case,
         pipeline=Pipeline.MIP_DNA,
     )
 
-    return mip_case
+    return mip_dna_case
+
+
+@pytest.fixture(name="mip_rna_case")
+def fixture_mip_rna_case(mip_rna_context, helpers) -> Family:
+    """Return a MIP RNA case."""
+
+    store = mip_rna_context.status_db
+
+    mip_rna_case = helpers.add_case(
+        store=store,
+        internal_id="mip-rna-case",
+        name="mip-rna-case",
```

Comment:
I think "mip-rna-case" should be test fixture to standardize this to allow for easier reuse later and avoid  hard coding

Reply:
It is not clear to me what I need to fix here @henrikstranneheim

Reply:
fixed with Vincent